hypha/static_src/src/sass/apply/components/_data-block.scss

```diff
@@ -25,22 +24,21 @@
     }
 
     &__title {
-        font-size: map-get($font-sizes, delta);
+        font-size: 20px;
+        font-weight: 700;
```

Comment:
Please use variables from `_variables.scss` instead of manual values for size and weight. If a value is missing add it to the variables instead.